api/src/main/java/org/openmrs/module/openconceptlab/updater/Importer.java

```diff
@@ -250,8 +261,14 @@ private void changeDuplicateNamesToIndexTerms(Concept concept, List<String> reso
 	 * @should add concept mapping and term
 	 * @should add concept mapping and unretire term
 	 * @should remove concept mapping and retire term
+	 * @should update mapping only if it has been updated since last import
 	 */
 	public Item importMapping(CacheService cacheService, Update update, OclMapping oclMapping) {
+		Item oldMappingItem = updateService.getLastSuccessfulItemByUrl(oclMapping.getUrl());
+		if(oldMappingItem!= null && !isMappingUpdatedSince(oldMappingItem, oclMapping)){
+			return new Item(update, oclMapping, ItemState.ALREADY_UP_TO_DATE);

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [An empty statement (semicolon) not part of a loop](https://www.codacy.com/app/agrzybkowski/openmrs-module-openconceptlab/file/2377174873/issues/source?bid=3218637&fileBranchId=3224807#l270)